wandb/sdk/wandb_init.py

```diff
@@ -364,8 +365,14 @@ def init(self):
             run._set_run_obj_offline(run_proto)
         else:
             ret = backend.interface.communicate_check_version()
-            if ret and ret.message:
-                wandb.termlog(ret.message)
+            if ret and ret.upgrade_message:
+                # if yanked, warn at header
+                if ret.yank_message:
+                    wandb.termlog(click.style(ret.yank_message, fg="red"))
+                wandb.termlog(ret.upgrade_message)
+                # if yanked, also warn at footer
+                if ret.yank_message:
+                    wandb.termlog(click.style(ret.yank_message, fg="red"))
```

Comment:
this isnt really the footer, this is just displaying it twice..   If you want to put this in the footer, then the info should be saved in a private field in the run object and displayed at:
wandb_run.py.Run._on_finish() (i think)

So if you want to do something in both the header and footer, I would:
- add: run._set_check_version_info(....)
- call: run._on_init()
- in wandb_run().  have something that prints out version info at header (_on_init() (Add this one), and _on_final())